### DISCUSSION TITLE
Moving threshold.conf and disabledsid.conf

### DISCUSSION THREAD

**Discussion by jdpadro** [Security-Onion-Solutions/securityonion#4339] [Unsupported Versions] [answered]
Prior to attempting the upgrade from 16.04 to 2.3.50 I decided to backup the following files, threshold.conf, disabledsid.conf amd tje bzar directory from within zeek. Once I completed the fresh install of 2.3.50 and subsequently upgrade (soup) I wanted to move those saved files and directory onto the new master. I attempted to look for them but they are not there. So question how do I add those files back into the master so they are passed onto the sensor whenever I can get those sensors installed and configured. In addition I also need to be able to tweak the suricata configuration to enable eve.json so as to pass alerts over to our Splunk environment as it was when I was still on 16.04. 

Thanks

**Comment by presianbg** [comment, marked as answer]
Hi,

I haven't used the old SO (aka 1) and I'm not sure if it used the YAML format.
The current version of SO (aka 2) is using pillar files with YAML for every node and you have to place your thresholding rules there:
https://docs.securityonion.net/en/latest/managing-alerts.html?highlight=threshold#threshold

You may use the idstools or the `so-rule` script to disable rules:
https://docs.securityonion.net/en/latest/managing-alerts.html?highlight=disable#disable-the-sid
or similar to the thresholds, use the pillar files:
![image](https://user-images.githubusercontent.com/2163486/120186077-b31f2380-c21b-11eb-81f4-ac1da2681e17.png)
 
https://docs.securityonion.net/en/latest/managing-alerts.html?highlight=disable#disable-the-sid

**Reply by presianbg** [reply]
Also not sure what you mean by tweaking the suricata to enable `eve.json`. 
Every sensor configure with suricata IDS will place its alerts in `/nsm/suricata/eve-yyyy-mm-dd-hh:mm.json` and from there you can pickup the log with Splunk forwarder and ship it to Splunk. 
By default the logs are being rotated every hour.

**Reply by jdpadro** [reply]
Thank you for letting me know that now the native suricata alerting is passed as a eve.json log. This does make it easy since I will no longer need to edit that file on every sensor. Good to know. Now if I can only figure out why  may master install using the 2.3.50 ISO failed to create the required /root/manager_setup/ directory required as part of the sensor installation. 

Thanks again
